server/models/models.js

```diff
@@ -8,7 +8,7 @@ const mongoose = require('mongoose');
 
 (async function () {
   try {
-    await mongoose.connect('mongodb://localhost/aggregate', { useMongoClient: true });
+    await mongoose.connect('mongodb://mongo:27017/aggregate', { useMongoClient: true });
```

Comment:
could we convert this to an env var so it can be changed to which env you are using